examples/non-fungible/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt::{Display, Formatter};
+
+use async_graphql::{Request, Response, SimpleObject};
+use fungible::Account;
+use linera_sdk::{
+    base::{AccountOwner, ApplicationId, ChainId, ContractAbi, ServiceAbi},
+    graphql::GraphQLMutationRoot,
+};
+use serde::{Deserialize, Serialize};
+
+pub type TokenId = String;
```

Comment:
I'm wondering if it would make sense to avoid using a `String` and `base64` in the contract. That could save some fuel. We can use the service to convert a token ID in bytes to a base64-encoded string in the service when handling a `mutation` :thinking:

Reply:
The problem with keeping `TokenId` as just a `Vec<u8>` is that I'll have to implement `Display` for it, which I can't because of the Orphan Rule :/ 
Lmk if I'm missing something

Reply:
I think we could do `struct TokenId(Vec<u8>)` , and that would allow implementing `Display` :thinking:

Reply:
I'd generally recommend using public type aliases `pub type .. =` very scarcely.